Sample/SextantSample.iOS/SextantSample.iOS.csproj

```diff
@@ -1,178 +1,178 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What actually changed in this file?  Just formatting?

Reply:
Ah, I wouldn't have touched iOS... so something Visual Studio did.  Sorry, I need to pore through the changes before I commit them.